### DISCUSSION TITLE
How to solve Notification issue

### DISCUSSION THREAD

**Discussion by Web95Dev** [Web95Dev/Infy-Invoice#2] [Q&A] [answered]
I could't fix an issue that socket.io doesn't work well for notification on this project. How to fix it?

**Comment by binary3oul** [comment, marked as answer]
Updated server.js

var https = require('https')
var app = express()

https.createServer({
key: fs.readFileSync('server.key'),
cert: fs.readFileSync('server.cert')
}, app)
.listen(8005, function () {
console.log('Example app listening on port 8005! Go to https://domen.com:3000/')
})

i think you should try first with http, if your code works well then you can upgrade to https

var io = require('socket.io')(http);
http.listen(8005, function () {
console.log('Listening to port 8005');
});
in client side :

let ip_address = 'http://domain.com/';
